.travis.yml

```diff
@@ -0,0 +1,16 @@
+language: node_js
+node_js:
+  - "12"
+  - "10"
+  - "8"
+os:
+  - linux
+  - osx
+cache:
+  yarn: true
+  directories:
+    - 'node_modules'
+    - packages/*/node_modules
+script:
+  - yarn lerna run build
```

Comment:
With the `"prepare"` script I mention below you shouldn't need a custom script for travis.